src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -219,6 +220,36 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
     action_type = CLIArgumentType(
         help='The action of virtual network rule. Possible value is Allow.'
     )
+
+    enable_alw_type = CLIArgumentType(
+        arg_type=get_three_state_flag(), min_api='2021-06-01',
+        help='The account level immutability property. The property is immutable and can only be set to true '
+             'at the account creation time. When set to true, it enables object level immutability for all '
+             'the containers in the account by default.'
+    )
```

Comment:
Have you tried that once set this property to true, customers are not allowed to turn it off? I just checked the SDK and didn't find the restriction.

Reply:
If it's false on creation, should we allow customer to update to true?